src/main/java/frc/robot/subsystems/TelescopeSubsystem.java

```diff
@@ -36,7 +40,21 @@ public TelescopeSubsystem() {
         armMotor.setSelectedSensorPosition(0);
 
         currentSetpoint = getCurrentPosition();
-        register();
+
+        if (Constants.IOConstants.LOGGING) {
+
+            final ShuffleboardTab telescopeTab = Shuffleboard.getTab("Telescope");
+
+            ShuffleboardLayout positionLayout = telescopeTab.getLayout("Position", BuiltInLayouts.kList);
+
+            positionLayout.addNumber("Current Angle", () -> getCurrentPosition());
+            positionLayout.addNumber("Setpoint", () -> getSetpoint());
+
+            ShuffleboardLayout resetLayout = telescopeTab.getLayout("Reset", BuiltInLayouts.kList);
+
+            resetLayout.addBoolean("Check Velocity", () -> checkVelocity());
```

Comment:
Maybe make this more clear that it's coming from the debouncer/what exactly this value is?